src/js/entities-service/entities/groups.js

```diff
@@ -11,6 +13,45 @@ const _Model = BaseModel.extend({
   getClinicians() {
     return Radio.request('entities', 'clinicians:collection', this.get('_clinicians'));
   },
+  addClinician(clinician) {
+    const url = `/api/groups/${ this.id }/relationships/clinicians`;
+    const groups = clinician.get('_groups') || [];
+
+    clinician.set({ _groups: groups.concat({ id: this.id }) });
```

Comment:
same here.. I don't think and existing groups array will get a change event on this line

Reply:
bah, forgot I was doing that in two places. Thanks